client/src/components/Payout.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { PlaidLink } from 'react-plaid-link';
+import Select from 'react-select';
+
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Paper from '@material-ui/core/Paper';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import styles from '../app.styles';
+import {
+  notify_error,
+  notify_success,
+  _fetch,
+} from '../common.js';
+
+class Payout extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      classes: props.classes,
+      global: props.global
+    };
+  }
+
+  onSuccess = async (token, metadata) => {
+    const { global } = this.state;
+    try {
+      // send token to server
+      await _fetch(global, '/payout/account/token/exchange', 'POST', {
+        token,
+        account_id: metadata.account.id,
+      });
+      notify_success('Linked account');
+    } catch (e) {
+      notify_error(e, 'Unable to link account.');
+    }
+  };
+
+  render() {
+    const { classes } = this.state;
+
+    return (
+      <main className={classes.main}>
+        <CssBaseline />
+        <Paper className={classes.paper}>
+          <PlaidLink
+            clientName="HelloVoter"
+            env="sandbox"
```

Comment:
Plaid sandbox environment. This needs to match the `plaid.environments.sandbox` in `payout.js`